arch/arm/include/armv7-a/irq.h

```diff
@@ -500,23 +500,15 @@ static inline_function uint32_t up_getsp(void)
 noinstrument_function
 static inline_function uint32_t *up_current_regs(void)
 {
-  uint32_t *regs;
-  __asm__ __volatile__
-  (
-    "mrc " "p15, " "0" ", %0, " "c13" ", " "c0" ", " "4" "\n"
-    : "=r"(regs)
-  );
-  return regs;
+  ARM_ISB();
```

Comment:
could you point out which document require to add barrier for TPIDRPRW read?

Reply:
I just want to add the volatile attribute, it looks like it's already included in the CP15 operation, let me revert